src/tools/base/BaseTool.js

```diff
@@ -26,6 +26,7 @@ class BaseTool {
     configuration,
     supportedInteractionTypes,
     mixins,
+    svgCursor,
```

Comment:
I think this will be the first `class` we pass in as part of configuration. Shouldn't be a big deal, but it might add a hurdle for anyone wanting to serialize config. I double checked, but we should be good with the current `globalSync` logic here:

https://github.com/cornerstonejs/cornerstoneTools/blob/master/src/store/addTool.js#L62-L82

Reply:
I originally passed a string, but it made more sense from the a plugin dev point of view to pass the class instance.

Also the `svgCursor` is just an object. Classes are just syntactic sugar in ES6.